src/abacus-ts/ontology.ts

```diff
@@ -0,0 +1,98 @@
+import { type RootState } from '@/state/_store';
+import { getCurrentMarketId } from '@/state/perpetualsSelectors';
+
+import {
+  getCurrentMarketAccountFills,
+  selectAccountFills,
+  selectAccountFillsLoading,
+  selectAccountOrdersLoading,
+  selectCurrentMarketOpenOrders,
+  selectCurrentMarketOrderHistory,
+  selectOpenOrders,
+  selectOrderHistory,
+  selectParentSubaccountOpenPositions,
+  selectParentSubaccountOpenPositionsLoading,
+  selectParentSubaccountSummary,
+  selectParentSubaccountSummaryLoading,
+} from './selectors/account';
+import {
+  selectAllAssetsInfo,
+  selectAllAssetsInfoLoading,
+  selectCurrentMarketAssetInfo,
+} from './selectors/assets';
+import {
+  selectRawIndexerHeightData,
+  selectRawIndexerHeightDataLoading,
+  selectRawValidatorHeightData,
+  selectRawValidatorHeightDataLoading,
+} from './selectors/base';
+import {
+  selectAllMarketsInfo,
+  selectAllMarketsInfoLoading,
+  selectCurrentMarketInfo,
+} from './selectors/markets';
+
+// every leaf is a selector or a paramaterized selector
+type NestedSelectors = {
+  [K: string]:
+    | NestedSelectors
+    | ((state: RootState) => any)
+    | (() => (state: RootState, ...other: any[]) => any);
+};
+
+// all data should be accessed via selectrs in this file
+// no files outside abacus-ts should access anything within abacus-ts except this file
+export const MegalodonCore = {
```

Comment:
this is meant to be THE place you look up the basic data for displaying the app, the authoritative source, the root of all other selectors

Reply:
one downside is that this will break tree shaking for everyone who uses this object :/